modules/oe_content_publication/config/install/core.entity_form_display.node.oe_publication.default.yml

```diff
@@ -94,39 +137,99 @@ content:
     third_party_settings: {  }
     type: entity_reference_autocomplete
     region: content
+  oe_publication_contact:
```

Comment:
We don't need oe_publication_contact anymore.

Reply:
This is removed.